### PR TITLE
Fix for charset issue when getting data from TXXX frame.

### PR DESCRIPTION
**Pull request by curtismenmuir**
- userDefinedData was being returned still encoded 
  (E.G. `"data":{"user_description":"LICENCE TYPE","data":"ÿþO"}}`
- Added missing charset into getStringWithCharsetAt() function
- Test updates

### CODE REVIEW

**Comment by aadsm** [comment]
Thank you so much for looking into this and putting this up! I’ll take a look over the weekend.

**Comment by aadsm** [comment]
Nice catch! Can you just fix the indentation? The project uses 2 spaces per indent and this diff changed all the touched files to 4 spaces.

**Comment by curtismenmuir** [comment]
@aadsm sorry I was supposed to go back and fix this. Thanks for taking the time to review this! :) I have pushed update to fix this indentation

**Comment by curtismenmuir** [comment]
@aadsm is the formatting ok now for getting this merged?

File: src/__tests__/ID3v2FrameReader-test.js [hunk 1]
```diff
@@ -338,13 +338,13 @@ describe("ID3v2FrameReader", function() {
       bin("ID2"), [0x00], // child 1
       bin("ID3"), [0x00], // child 2
       bin("TIT1"), // child 1
-        [0x00, 0x00, 0x00, 0x02], // size
-        [0x00, 0x00], // flags
-        [0x00, 0x00], // text encoding + null terminated string
+      [0x00, 0x00, 0x00, 0x02], // size
+      [0x00, 0x00], // flags
+      [0x00, 0x00], // text encoding + null terminated string
```
Comment (aadsm) [inline src/__tests__/ID3v2FrameReader-test.js]: I actually had these indent so you could visually see they belong to the TIT1.

Comment (curtismenmuir) [reply src/__tests__/ID3v2FrameReader-test.js]: Will add space back in

**Comment by curtismenmuir** [comment]
@aadsm thanks for reviewing. No worries about taking some time! :) I have removed package-lock.json updates and I have fixed the formatting in the tests!

**Comment by curtismenmuir** [comment]
@aadsm do those updates look ok now?

**Comment by curtismenmuir** [comment]
@aadsm thanks for merging this :) When are you planning on releasing a new version with the fix? I have a release coming at the end of the month for a project I am working on and it would be great to have the fix included.

**Comment by aadsm** [comment]
@curtismenmuir no problem, I'll do that over the weekend.

**Comment by curtismenmuir** [comment]
@aadsm that would be perfect, thank you so much!

**Comment by aadsm** [comment]
@curtismenmuir fyi: I've just published a new version with your fixes :) I also added you to the contributors list: https://github.com/aadsm/jsmediatags/blob/master/CONTRIBUTORS.md#bugs.

**Pull request merged**